.travis.yml

```diff
@@ -33,29 +34,31 @@ env:
 matrix:
   fast_finish: true
   include:
-    - php: '7.0'
+    - php: '7.1'
       env: TARGET=docs
-    - php: '7.0'
+    - php: '7.1'
       env: TARGET=lint
     - php: '5.6'
       env: COMPOSER_FLAGS="--prefer-lowest"
-    - php: '7.0'
+    - php: '7.1'
       env: SYMFONY=2.8.*
-    - php: '7.0'
+    - php: '7.1'
+      env: SYMFONY=3.0.*
```

Comment:
I don't see how this makes sense. 7.0 is still present in the dev-kit config, why would it be removed?

But I think you can safely merge. Whatever the reason for this it, we can figure it out later and keep moving.